src/app/profiles.rs

```diff
@@ -159,7 +159,14 @@ where
                         .routes
                         .into_iter()
                         .filter_map(move |orig| convert_route(orig, retry_budget.as_ref()));
-                    match tx.start_send(routes.collect()) {
+                    let overrides = profile
+                        .dst_overrides
+                        .into_iter()
+                        .filter_map(|orig| convert_dst_override(orig));
```

Comment:
Nit/TIOLI: This can just be
```suggestion
                        .filter_map(convert_dst_override);
```